src/browser/themes/pulse/sidebar_tabs.css

```diff
@@ -111,6 +111,7 @@
 
 #sidebar-icons {
   overflow: scroll;
```

Comment:
Why don't you use `overflow: auto;`?

Reply:
You are correct I think I must have written scroll at that moment, although because the scrollbars are hidden anyway it shouldn't cause any problems. Unfortunately the scrollbars are not the most aesthetically pleasing.